src/TypeScript/EslintBridgeClient/EslintBridgeClient.cs

```diff
@@ -93,7 +98,15 @@ private Task Close()
 
         public async void Dispose()
         {
-            await Close();
+            try
```

Comment:
As we discussed in a previous PR, JavascriptAnalyzer will dispose this class when the eslint-bridge's port changes. This will then attempt to call Close on process that is no longer running, fail, and leave httpWrapper behind.

Reply:
Right now it's after the catch 👍